pennylane/pauli/dla/center.py

```diff
@@ -56,23 +96,90 @@ def center(
     >>> qml.center(g)
     [X(0)]
 
+    .. details::
+        :title: Derivation
+        :href: derivation
+
+        The center :math:`\mathfrak{z}(\mathfrak{k})` of an algebra :math:`\mathfrak{k}`
+        can be computed in the following steps. First, compute the
+        :func:`~.pennylane.structure_constants`, or adjoint representation, of the algebra
+        with respect to some basis :math:`\mathbb{B}` of :math:`\mathfrak{k}`.
+        The center of :math:`\mathfrak{k}` is then given by
+
+        .. math::
+
+            \mathfrak{z}(\mathfrak{k}) = \operatorname{span}\left\{\bigcap_{x\in\mathbb{B}}
+            \operatorname{ker}(\operatorname{ad}_x)\right\},
+
+        i.e., the intersection of the kernels, or null spaces, of all basis elements in the
+        adjoint representation.
+
+        The kernel can be computed with ``scipy.linalg.null_space``, and vector space
+        intersections are computed recursively from pairwise intersections. The intersection
+        between two vectors spaces :math:`V_1` and :math:`V_2` given by (orthonormal) bases
+        :math:`\mathbb{B}_i` can be computed from
+        :math:`\operatorname{ker}([\mathbb{B}_1 | -\mathbb{B}_2])`. For an (orthonormal)
```

Comment:
In the rendered version this kind of looks like $\text{ker}(|B_1| - B_2)$

![image](https://github.com/user-attachments/assets/434e7f31-c85a-4377-976f-56a4bc0a83a0)

not sure but perhaps this helps

```suggestion
        :math:`\operatorname{ker}(\left[\mathbb{B}_1 | -\mathbb{B}_2\right])`. For an (orthonormal)
```

Alternatively, make the vline big and the square angles small, or just round angles ^^

Reply:
unrelated: I am also not sure I get the the notation $\{(u_1^{(i)}, u_2^{(i)})^T\}_i$ what is going on here?

Reply:
They are supposed to be two stacked column vectors, not sure how to denote this best. I could actually display it as $\left(\begin{array}{c} u^{(i)}_1 \\ u^{(i)}_2 \end{array}\right)$, but maybe that will destroy the text appearance a bit too much?

Reply:
I actually skipped writing this out in math and described it. Let me know what you think.

Regarding the formatting of the h-stacked bases, I used your tip and inserted spaces, hope that helps.